controls/docker_host_os_level1.rb

```diff
@@ -121,11 +135,17 @@
   impact 1.0
   title 'Audit docker daemon'
   desc 'Apart from auditing your regular Linux file system and system calls, audit Docker daemon as well. Docker daemon runs with \'root\' privileges. It is thus necessary to audit its activities and usage.'
+  tag 'host'
   ref 'https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Security_Guide/chap-system_auditing.html'
 
   describe auditd_rules do
     its(:lines) { should include('-w /usr/bin/docker -p rwxa -k docker') }
   end
+    describe service('auditd') do
```

Comment:
yes i agree, i test now with debian and will extened this to the other operating systems